src/app/Main/Body/QueryPage/Drawer/Drawer-Layout.component.tsx

```diff
@@ -43,6 +44,7 @@ export interface DrawerLayoutProps {
     drawerWidth: number;
     dataUniqueValues: FilterInterface;
     filterInfo: FilterContextInterface;
+    subFilters: ClientSingleFilterConfig[];
```

Comment:
Not sure if it wouldn't make more sense to have the subfilters nested within filterInfo somehow.  After all subfilters by themselves can't really exist and only make sense in the context of main filters,  This would require a heavy rewrite of the FilterContextInterface I guess.

Reply:
Yes, I get the point. That was my first thought too. But the information "Which filters are subfilters?" is only fetched once from the server when the component is loaded. But every change of the filters/filterInfo (which then also include the subfilters) generates a new call to the Isolate API. React doesn't like to store these two things in one hook (or I haven't found the right way yet).

Reply:
Another solution would be to hardcode the subfilters in the frontend, then I wouldn't need the subfilter-info from the api (parent/target) at all. Hard-coded stuff can also be saved in the filter hook. And since the translation takes place in the frontend, they are already hard-coded anyway.